### PR TITLE
Extract javascript from profile page into js/profile.js

### PR DESCRIPTION
**Pull request by XtheWiz**
Could you please review this change?

### CODE REVIEW

**Comment by XtheWiz** [comment]
> @XtheWiz fix the bug in line where you are including js file. rest is okay 👍

Could you please give me a hint of this bug ?

File: Projects/templates/Projects/profile.html [hunk 1]
```diff
@@ -153,118 +153,6 @@ <h3 style="color: white"> You are a mentor and None of your issues have been att
...
-            $(".prs_pending").hide();
-            $(".issues_all").hide();
-            $(".prs_vclosed").hide();
-            $(".prs_unvclosed").hide();
-        });
-
-
-    })      
-    </script>
+<script type="text/javascript" src="js/profile.js"></script>
```
Comment (dbads) [inline Projects/templates/Projects/profile.html]: fix src here?

**Comment by dbads** [comment]
> > @XtheWiz fix the bug in line where you are including js file. rest is okay +1
> 
> Could you please give me a hint of this bug ?

yeah, I already mentioned it. you have to just fix the src of js file you are adding.

**Comment by XtheWiz** [comment]
> > > @XtheWiz fix the bug in line where you are including js file. rest is okay +1
> > 
> > 
> > Could you please give me a hint of this bug ?
> 
> yeah, I already mentioned it. you have to just fix the src of js file you are adding. It should not be hardcoded.

Thank you very much, I already fixed and push it again, could you please review it?

File: Projects/templates/Projects/profile.html [hunk 2]
```diff
@@ -153,118 +153,6 @@ <h3 style="color: white"> You are a mentor and None of your issues have been att
...
-            $(".prs_pending").hide();
-            $(".issues_all").hide();
-            $(".prs_vclosed").hide();
-            $(".prs_unvclosed").hide();
-        });
-
-
-    })      
-    </script>
+<script type="text/javascript" src="static/Projects/js/profile.js"></script>
```
Comment (dbads) [inline Projects/templates/Projects/profile.html]: Though it'll work here you shouldn't hard code the path.
it should be src="{% static 'Projects/js/profile.js' %}"

**Pull request merged**